modules/text2vec-ollama/clients/ollama.go

```diff
@@ -85,11 +105,11 @@ func (v *ollama) vectorize(ctx context.Context, input string,
 }
 
 func (v *ollama) parseEmbeddingsResponse(statusCode int,
-	bodyBytes []byte, input string,
-) (*ent.VectorizationResult, error) {
+	bodyBytes []byte, input []string,
+) (*modulecomponents.VectorizationResult, error) {
 	var resBody embeddingsResponse
 	if err := json.Unmarshal(bodyBytes, &resBody); err != nil {
-		return nil, errors.Wrap(err, fmt.Sprintf("unmarshal response body. Got: %v", string(bodyBytes)))
+		return nil, fmt.Errorf("unmarshal response body. Got: %v: %w", string(bodyBytes), err)
```

Comment:
please use wrapf instead. You loose the error-type with Errorf. Probably doesn't matter here, but that is a good idea in general